.github/workflows/check_cli.yml

```diff
@@ -0,0 +1,30 @@
+name: Check CLI
```

Comment:
since we're actively working on the CLI, I think it'll be useful to have extensive tests...is this a suitable frequency/setup to test these? @tbirdso @jjomier